_posts/2020-04-17-news5.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: Tóm tắt buổi họp báo của thủ tướng Abe ngày 17/4/2020
+category: thematic
+---
+
+**Biên soạn:**  
+**Đoàn Lê Hải Ngọc, nhân viên công ty Tokyu**  
+**Tạ Thị Khánh Hòa, kiểm toán viên Công ty PwC**  
+**Hiệu đính:**  
+**Phạm Anh Phong**
+
+
+### Kêu gọi toàn dân giảm tiếp xúc xã hội
+
+・Đã có số liệu số người đi lại xung quanh ga Shibuya giảm 60%, ga Umeda giảm 70% so với trước đợt đại dịch.  
+・Tuy nhiên cả nước vẫn chưa đạt được mục tiêu giảm tiếp xúc giữa người với người tối thiểu 70%, khuyến khích 80% như đã đặt ra.  
+・Đã có hơn 200 ca nhiễm tại Tokyo ngày hôm nay, con số vượt mức kỷ lục từ trước tới giờ. Tổng số ca nhiễm tại thời điểm ngày hôm nay ở Tokyo đạt 3,000 ca, Osaka 1,000 ca.  
+・Thủ tướng yêu cầu toàn dân giảm tiếp xúc xã hội để cứu chính những người thân thiết của họ.  
+
+### Nêu lý do tuyên bố tình trạng khẩn cấp trên toàn nước
+
+・Theo phân tích của các nhà chuyên môn, có rất nhiều ổ dịch được báo cáo ở các tỉnh thành trên toàn quốc. Nguyên nhân của hiện tượng này chính là ở việc di chuyển từ vùng thành phố về các tỉnh vào 3 ngày nghỉ lễ của tháng 3.  
+・Để phòng chống sự bùng nổ dịch ở các địa phương có nhiều người cao tuổi cư trú trong tuần lễ Vàng, Chính phủ đã mở rộng phạm vi tỉnh tuyên bố tình trạng khẩn cấp trên toàn quốc. Thời hạn của tình trạng khẩn cấp vẫn giữ nguyên, là đến ngày 6/5/2020.  
```

Comment:
`Chính phủ đã mở rộng phạm vi tỉnh tuyên bố tình trạng khẩn cấp trên toàn quốc`

Câu này hơi khó hiểu. Suggest bỏ chữ "tỉnh" hoặc đơn giản là "Chính phủ đã tuyên bố tình trạng khẩn cấp trên toàn quốc"

Reply:
e bỏ chữ "tỉnh" rồi ạ